test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorFactoryTest.cs

```diff
@@ -246,14 +425,14 @@ public void CreateDescriptor_ResolvesMultipleTagHelperDescriptorsFromSingleType(
             var validProp = typeof(MultiTagTagHelper).GetProperty(nameof(MultiTagTagHelper.ValidAttribute));
             var expectedDescriptors = new[] {
                 new TagHelperDescriptor(
-                    "div",
+                    "p",
                     typeof(MultiTagTagHelper).FullName,
                     AssemblyName,
                     new[] {
                         new TagHelperAttributeDescriptor("valid-attribute", validProp)
                     }),
                 new TagHelperDescriptor(
-                    "p",
+                    "div",
```

Comment:
Order used to be controlled by the order of `HtmlElementName` attributes returned via reflection, now it's controlled by `string.Split` :smile: